src/Algebra/Structures.agda

```diff
@@ -460,8 +460,6 @@ record IsCancellativeCommutativeSemiring (+ * : Op₂ A) (0# 1# : A) : Set (a 
 
   open IsCommutativeSemiring isCommutativeSemiring public
 
-
-
```

Comment:
Please do not unnecessarily delete whitespace.

Reply:
Sorry, I do not understand what is wrong there.

Reply:
You have deleted empty lines for no reason, in a file you have changed nothing in. Please do not change files unnecessarily.

Reply:
I copied this module back from to-day's _master_.